python/gps/agent/ros/agent_ros.py

```diff
@@ -7,10 +9,12 @@
 from gps.agent.agent_utils import generate_noise, setup
 from gps.agent.config import AGENT_ROS
 from gps.agent.ros.ros_utils import ServiceEmulator, msg_to_sample, \
-        policy_to_msg
+        policy_to_msg, tf_policy_to_action_msg, tf_obs_msg_to_numpy
 from gps.proto.gps_pb2 import TRIAL_ARM, AUXILIARY_ARM
 from gps_agent_pkg.msg import TrialCommand, SampleResult, PositionCommand, \
-        RelaxCommand, DataRequest
+        RelaxCommand, DataRequest, TfActionCommand, TfObsData
+from gps.algorithm.policy.tf_policy import TfPolicy
```

Comment:
This import probably requires tensorflow to be installed, which is bad if someone wants to use AgentROS w/o tensorflow installed. Can you make sure that the examples all still run without tensorflow installed? (You can use a try/catch if needed)